crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -32,6 +32,13 @@ fn run() -> proc_exit::ExitResult {
 
     init_logging(args.verbose.log_level());
 
+    // https://github.com/console-rs/dialoguer/issues/294
+    ctrlc::set_handler(move || {
+        let _ = console::Term::stdout().show_cursor();
+        std::process::exit(0);
```

Comment:
Is `exit(0)` the correct way of exiting on ctrl-c to mirror behavior before this change?

Reply:
You're right, the previous exit code for `ctrl-c` was 130, and according to my quick research 130 is preferred over 0 when `ctrl-c` killed the process via SIGTERM.